CONTRIBUTING.md

```diff
@@ -1,9 +1,8 @@
 # Contributing Guide
 
 ## Contributor License Agreement (CLA)
-The CLA forms are currently unique to this app.
 Sign and email the appropriate Contributor License agreement below to GSFC-SoftwareRelease@mail.nasa.gov and copy cfs-program@lists.nasa.gov
-* Corporate Contributor License agreement : https://github.com/nasa/MD/blob/main/docs/Corp_CLA_MD_240.pdf
-* Individual Contributor License agreement : https://github.com/nasa/MD/blob/main/docs/Indv_CLA_MD_240.pdf
+* cFS Apps Corporate Contributor License agreement : https://github.com/nasa/cFS/blob/main/cFS_Apps_Corporate_CLA.pdf
+* cFS Apps Individual Contributor License agreement : https://github.com/nasa/cFS/blob/main/cFS_Apps_Individual_CLA.pdf
```

Comment:
I recommend removing this since it's just a repeat of what's in the top level contributing guide.